webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -817,7 +818,7 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
               />
             </ComparisonView>
             {showPagination && (
-              <Columns>
+              <Row>
                 <Column align="right">
```

Comment:
fix: alignment

Reply:
These buttons should be aligned to the right side of the screen
![Screenshot 2023-11-14 at 16-31-21 Uncategorized Experiments - Determined](https://github.com/determined-ai/determined/assets/15078396/6056c0a2-1844-4349-82f6-24ed14270d3a)